pkg/virt-launcher/virtwrap/manager.go

```diff
@@ -320,34 +324,61 @@ func (l *LibvirtDomainManager) FinalizeVirtualMachineMigration(vmi *v1.VirtualMa
 	return l.finalizeMigrationTarget(vmi)
 }
 
-// HotplugHostDevices attach host-devices to running domain
-// Currently only SRIOV host-devices are supported
+// HotplugHostDevices attach host-devices to running domain, currently only SRIOV host-devices are supported.
+// This is operation runs on the background each at a time.
 func (l *LibvirtDomainManager) HotplugHostDevices(vmi *v1.VirtualMachineInstance) error {
```

Comment:
It seems that this method always returns nil. Is it necessary to keep the returned value?

Reply:
You right, now with the new changes it does returns an error

Reply:
"only one hotplug operation can occur at a time"

Reply:
Done